lib/handlers/authenticate-handler.js

```diff
@@ -139,7 +139,7 @@ class AuthenticateHandler {
 
   getTokenFromRequestHeader (request) {
     const token = request.get('Authorization');
-    const matches = token.match(/^Bearer\s(\S+)/);
+    const matches = token.match(/^Bearer\s([0-9a-zA-Z-._~+/]+=*)$/);
```

Comment:
This regex would allow a tab or newline after `Bearer` due to `\s` class treating tabs and newlines as whitespace  (I missed that in the other PR #235 but I did not merge yet). I would propose to use 

```js
/^Bearer ([0-9a-zA-Z-._~+/]+=*)$/
```

where an explicit space character is placed after `Bearer` instead of `\s`